department-of-veterans-affairs/caseflow
# CAMO users can indicate that an appeal is recommended for cancellation

Resolves [#{2140}](https://vajira.max.gov/browse/CASEFLOW-2140)

### Description
Added logic to grab previous Program Office Notes from previously completed AssessDocumentationTasks and displayed them at the top of the Send to Board Intake Modal as well as in the Case Details Timeline of the appeal.

### Acceptance Criteria

### Testing Plan
1. Add the `vha_irregular_appeals` feature toggle
2. Go to Case Details
3. Click Available actions dropdown
4. Select "Send to Board Intake"
5. If there are notes from the Program Office you will see them at the top of the modal.
6. Select a radio option and enter notes in the text field
7. Click "Submit"
8. Navigate to the VhaCamo Completed tab and locate appeal
9. Click on appeal to access the Case Details page
10. Navigate down to the Case Timeline and open the Task Instructions from the most recent completed task.
11. Verify Instructions are formatted correctly

### User Facing Changes

 BEFORE|AFTER
 
![image](https://user-images.githubusercontent.com/69862429/149379719-b48fd613-a96d-4dfd-a9b5-e5f761cc8466.png)
|
![image](https://user-images.githubusercontent.com/69862429/149380238-33f76d12-266d-4740-9f5b-28ee023af0fb.png)
![image](https://user-images.githubusercontent.com/69862429/149380459-c72890e4-84f8-4d92-8ec4-0775dec85f65.png)
